php/example_code/dynamodb/Readme.md

```diff
@@ -0,0 +1,84 @@
+# Amazon DynamoDB code examples for the AWS SDK for PHP v3
+
+## Purpose
+
+Shows how to use the AWS SDK for PHP v3 to get started using operations in Amazon DynamoDB. Learn to create tables, add
+items, update data, create custom queries, and delete tables and items.
+
+*Amazon DynamoDB is a fully managed NoSQL database service that provides fast and predictable performance with seamless
+scalability.*
+
+## ⚠️ Important
+* Running this code might result in charges to your AWS account.
+* Running the tests might result in charges to your AWS account.
+* We recommend that you grant your code least privilege. 
+At most, grant only the minimum permissions required to perform the task. 
+For more information, see 
+[Grant least privilege](https://docs.aws.amazon.com/IAM/latest/UserGuide/best-practices.html#grant-least-privilege).
+* This code is not tested in every AWS Region. 
+For more information, see 
+[AWS Regional Services](https://aws.amazon.com/about-aws/global-infrastructure/regional-product-services).
+
+## Code examples
+
+### Single actions
+
+**Actions**
```

Comment:
If it makes sense to do so, I recommend using the header names and introductory sentences outlined in the [template wiki](https://github.com/awsdocs/aws-doc-sdk-examples/wiki/Service-level-README-template).


For example:

### Single actions 

Code excerpts that show you how to call individual service functions.

[Creating a table]... etc.

Reply:
Fixed.